test/internals/errors.test.js

```diff
@@ -88,9 +88,16 @@ test('Create error with different base', t => {
   const err = new NewError('dude')
   t.type(err, Error)
   t.type(err, TypeError)
-  t.equal(err.name, 'FastifyError [CODE]')
-  t.equal(err.message, 'CODE: hey dude')
+  t.equal(err.name, 'FastifyError')
+  t.equal(err.message, 'hey dude')
   t.equal(err.code, 'CODE')
   t.equal(err.statusCode, 500)
   t.ok(err.stack)
 })
+
+test('FastifyError.toString returns code', t => {
+  t.plan(1)
+  const NewError = createError('CODE', 'foo')
+  const err = new NewError()
+  t.equal(err.toString(), 'FastifyError [CODE]: foo')
```

Comment:
Does the default error serializer rely on toString?

Reply:
Yes.